include/merlin_hashtable.cuh

```diff
@@ -598,9 +599,19 @@ class HashTable {
 
       keys_not_empty<K>
           <<<grid_size, block_size, 0, stream>>>(evicted_keys, d_masks, n);
+
+      void* d_temp_storage = nullptr;
+      size_t temp_storage_bytes = 0;
+      cub::DeviceScan::ExclusiveSum(d_temp_storage, temp_storage_bytes,
```

Comment:
It would be better to add the `CUDA_CHECK` for these kernel calls.

Reply:
Accept